libs/watt/src/lib/foundations/typography/_typography.import.scss

```diff
@@ -49,6 +49,10 @@ $font-sizes: (
     "font-size": 14px,
     "line-height": 20px,
   ),
+  "xs": (
+    "font-size": 12px,
+    "line-height": 22px,
```

Comment:
line-height seems abit of compared to other font sizez. It seems to get smaller each time except for the last time? This time we increase by 2 ?

Reply:
I know, however the font-sizes are taken from Pernille's figma so alternatives are only using it for this use-case or not follow the sizes she has used.